packages/source-fetcher/lib/etherscan.ts

```diff
@@ -54,12 +56,14 @@ const EtherscanFetcher: FetcherConstructor = class EtherscanFetcher
       "rinkeby",
       "goerli",
       "optimistic",
-      "kovan-optimistic"
+      "kovan-optimistic",
+      "arbitrum",
+      "polygon"
```

Comment:
We should probably add the testnet versions as well.  Arbitrum's testnet etherscan is https://testnet.arbiscan.io/ and polygon's is  https://mumbai.polygonscan.com/

Reply:
Okay I wrote to you in Slack about this. Let me know if you think this should be included in this PR or if we should merge this so you can work on the bridge box and then we can create a separate PR to add those two.

Reply:
FWIW, my opinion is that it would be OK to save that for later.

Reply:
Agreed, this is not blocking and fine for later.